src/main/resources/application.properties

```diff
@@ -3,4 +3,5 @@ spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/electronicrefere
 spring.datasource.username=root
 spring.datasource.password=7CiUaX359m
```

Comment:
remove your password (replace with "root", "password" or smth like that)

Reply:
Done